src/main/java/fact/io/hdureader/BinTableReader.java

```diff
@@ -70,6 +72,14 @@ public static BinTableReader forBinTable(BinTable binTable){
             return  map;
         }
 
+        @Override
+        public void skipToRow(int num) throws IOException {
```

Comment:
docu?

Reply:
In the reader interface, should I copy it also here?